docs/readme.md

```diff
@@ -4,6 +4,10 @@
 
 Get GitHub for Visual Studio set up to bring the GitHub flow to Visual Studio. Authenticate to GitHub.com or GitHub Enterprise, keep the extension up-to-date, and review your preferred settings.
 
-### [Contributing to Projects with GitHub for Visual Studio](contributing/index.md)
+### [Using GitHub for Visual Studio](using/index.md)
 
-Use GitHub for Visual Studio to manage your projects and work with pull requests.
\ No newline at end of file
+Use GitHub for Visual Studio to manage your projects and work with pull requests.
+
+### [Developing GitHub for Visual Studio]
```

Comment:
Is this supposed to be a link to https://github.com/github/VisualStudio/tree/master/docs/developer?